gamedata.txt

```diff
@@ -0,0 +1,40 @@
+What kind of data does the frontend need?
+program counter for each process + current one
+we need to know the content of every cell at each point in time
+-> pretty sure we will be resorting to diffs / commits
+
+players enter programs
+interpreter runs entire game
+interpreter trackt changes per tick
+ - a "commit" contains all changes in this turn
+ - also which player made them
+ - checkout GameResult below
+result would be a big list of changes that build on top of each other and that describe the entire game and enable reconstructing the game-> look at GameResult below
+
+via return from shork.run to api backend
+serialized as a big json blob to the frontend
+goes directly as is to the frontend
+frontend is responsible for displaying everything "stupidly"
+
+GameResult:
+{
+    winner: as before
```

Comment:
If we already introduce a new gameresult object, I'd propose moving directly to something like
```json
{
  outcome: {
    kind: (WIN | DRAW),
    player?: String,
  },
  ...
}